### PR TITLE
Voting Contract

### PR DESCRIPTION
**Pull request by phu-tang**

### CODE REVIEW

File: contract/neo_voting.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from boa.interop.Neo.Action import RegisterAction
+from boa.blockchain.vm.System.ExecutionEngine import GetScriptContainer, GetExecutingScriptHash
+
+OnVote = RegisterAction('OnVote', 'kaiSmc', 'voter', 'candidate')
+
+
+def Main(operation, args):
+    if operation == "deposit":
```
Comment (DNK90) [inline contract/neo_voting.py]: why operation == "deposit"?

File: contract/neo_voting.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        if(len(args) != 2):
```
Comment (DNK90) [inline contract/neo_voting.py]: incorrect syntax, should be 
if len(args) != 2:

File: contract/neo_voting.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        candidateName = args[1];
+        tx = GetScriptContainer()
+        references = tx.References
+        reference = references[0]
+        voter = GetScriptHash(reference)
+        context = GetContext()
+
+        voterKey = kaiSmc.join(voter)
+
+        isVote = Get(context, key)
```
Comment (sontranrad) [inline contract/neo_voting.py]: what is key here ?

Comment (phu-tang) [reply contract/neo_voting.py]: typo: should be voterKey

**Pull request merged**